shell/src/main/java/alluxio/shell/command/LsCommand.java

```diff
@@ -64,12 +66,18 @@ public static String formatLsString(boolean acl, boolean isFolder, String permis
     } else {
       memoryState = inMemory ? STATE_FILE_IN_MEMORY : STATE_FILE_NOT_IN_MEMORY;
     }
+    String sizeStr = "";
```

Comment:
nit: You can use String sizeStr = humanReadable ? FormatUtils.getSizeFromBytes(size):String.valueOf(size);